homeassistant/components/automation/config.py

```diff
@@ -45,6 +47,7 @@
             CONF_ID: str,
             CONF_ALIAS: cv.string,
             vol.Optional(CONF_DESCRIPTION): cv.string,
+            vol.Optional(CONF_STORED_TRACES, default=STORED_TRACES): cv.positive_int,
```

Comment:
Instead of having a single setting like this, I think that we should have a `trace` key that can contain more information. 

```yaml
trace:
  xxxx: 3
```

Reply:
Fixed